drivers/SmartThings/sonos/src/api/sonos_connection.lua

```diff
@@ -93,14 +93,21 @@ local function _open_coordinator_socket(sonos_conn, household_id, self_player_id
 
   if coordinator_id ~= self_player_id then
     local coordinator = sonos_conn.driver.sonos:get_household(household_id).players[coordinator_id]
```

Comment:
I think the wider issue is that we should be nil-checking `sonos_conn.driver.sonos:get_household(household_id)` or at least forcing the return of an empty table.